### PR TITLE
Update packages-3rd streak

### PR DESCRIPTION
**Pull request by suhan-paradkar**
I have removed the patch for calcurse since it is no longer required

### CODE REVIEW

**Comment by Grimler91** [comment]
The TMPDIR patch for calcurse should probably be updated instead of removed, /tmp is hardcoded as fallback also in the new version: https://github.com/lfos/calcurse/blob/a49adf2db7cfb3e4b12fed948c76205a42f334cd/src/utils.c#L775

**Comment by suhan-paradkar** [comment]
> The TMPDIR patch for calcurse should probably be updated instead of removed, /tmp is hardcoded as fallback also in the new version: https://github.com/lfos/calcurse/blob/a49adf2db7cfb3e4b12fed948c76205a42f334cd/src/utils.c#L775

I am really bad at creating patches.. but i'll try to fix it...

**Comment by suhan-paradkar** [comment]
@Grimler91 I updated the patch..

File: packages/fix-hardcoded-paths.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -uNr calcurse-4.6.0/src/utils.c calcurse-4.6.0.mod/src/utils.c
+--- calcurse-4.6.0/src/utils.c	2020-02-04 23:46:34.000000000 +0200
++++ calcurse-4.6.0.mod/src/utils.c	2020-10-01 20:31:32.951590892 +0300
+@@ -775,7 +775,7 @@
+ 		return P_tmpdir;
```
Comment (xeffyr) [inline packages/fix-hardcoded-paths.patch:5]: Why that file appeared directly under "packages" directory? It should be inside "calcurse".

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/fix-hardcoded-paths.patch (hunk 1 above)
Comment (suhan-paradkar) [reply packages/fix-hardcoded-paths.patch:5]: Oh yeah.. my bad..

**Review by xeffyr — APPROVED** [review]

**Pull request merged**